app/controllers/search_controller.rb

```diff
@@ -119,11 +119,24 @@ def feed_content
   end
 
   def reactions
-    result = Search::ReadingList.search_documents(
-      params: reaction_params.to_h, user: current_user,
-    )
-
-    render json: { result: result["reactions"], total: result["total"] }
+    if FeatureFlag.enabled?(:search_2_reading_list)
+      result = Search::Postgres::ReadingList.search_documents(
+        current_user,
+        page: reaction_params[:page],
+        per_page: reaction_params[:per_page],
+        statuses: reaction_params[:status],
+        tags: reaction_params[:tag_names],
+        term: reaction_params[:search_fields],
+      )
```

Comment:
Recyling params as we want to change the frontend as little as possible, might simplify in the future

Reply:
I'd maybe leave this as a comment in the code.

Reply:
Agreed, I think a comment here would be useful :)

Reply:
Done